.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,33 @@
 {
     "dockerFile": "Dockerfile",
-    "postCreateCommand": "sh /setup.sh",
-    "extensions": ["ms-python.python"]
+    "customizations": {
```

Comment:
this looks like a mix of tabs and spaces, can we go all spaces?

Reply:
Done.